sonarts-core/tests/ruling/ruling.ts

```diff
@@ -54,7 +54,7 @@ export function runRules(rules: tslint.Rules.AbstractRule[], tsConfigFiles: stri
     files.forEach(file => {
       rules.forEach(Rule => {
         const rule = initRule(Rule);
-        const errorLines = runRuleOnProjectFile(rule, file, program);
+        const errorLines = runRuleOnProjectFile(rule, file, program).map(failure => failure.getStartPosition().getLineAndCharacter().line + 1);
```

Comment:
@inverno why this refactoring?

Reply:
@vilchik-elena I'm considering to replace `runRuleOnProjectFile` with `rules.executeRule`. This refactoring approached the two functions.